src/scmdata/run.py

```diff
@@ -1563,20 +1573,42 @@ def process_over(
         group_cols = list(set(ts.index.names) - set(cols))
         grouper = ts.groupby(group_cols)
 
-        if operation == "median":
-            res = grouper.median(**kwargs)
-        elif operation == "mean":
-            res = grouper.mean(**kwargs)
-        elif operation == "quantile":
-            res = grouper.quantile(**kwargs)
+        #  https://pandas.pydata.org/pandas-docs/stable/reference/groupby.html
+        allowed_pd_ops = [
+            "count",
+            "cumcount",
+            "cummax",
+            "cummin",
+            "cumprod",
+            "cumsum",
+            "first",
+            "last",
+            "max",
+            "mean",
+            "median",
+            "min",
+            "prod",
+            "rank",
+            "std",
+            "sum",
+            "var",
+            "quantile",
+        ]
+
+        if isinstance(operation, str):
+            if operation not in allowed_pd_ops:
+                raise ValueError("invalid process_over operation")
+            grouper_func = getattr(grouper, operation)
```

Comment:
Could we do something like 

```python
try:
    grouper_func = getattr(grouper, operation)
except AttributeError:
    raise ValueError("invalid process_over operation, see https://pandas.pydata.org/pandas-docs/stable/reference/groupby.html for available options")
```

rather than hard-coding available ops

Reply:
I had that as a first iteration, but not all ops can/should be available.

Reply:
nice, maybe add a note to that effect somewhere in the docs?